src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -227,6 +270,28 @@ public function setup(): void
             $this->driverConnection->getConfiguration()->setFilterSchemaAssetsExpression($assetFilter);
         }
 
+        if ($this->usePgsqlNotify) {
+            $sql = sprintf(<<<'SQL'
```

Comment:
wrap in begin...commit?

Reply:
What would be the benefit?

Reply:
is `exec` transactional? you want both statements to fail or both to succeed.

Reply:
I'm not sure, `pg_query` is transnational, but I cannot find the information for exec.